public_suffix_list.dat

```diff
@@ -1,4 +1,4 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
+2// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This would get immediately rejected, remove this 2

Reply:
I just edited, thank you